spectrocloud/schemas/scan_policy.go

```diff
@@ -0,0 +1,30 @@
+package schemas
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func ScanPolicySchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		MaxItems: 1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"configuration_scan_schedule": {
+					Type:        schema.TypeString,
+					Required:    true,
+					Description: "The schedule for configuration scan.",
+				},
+				"penetration_scan_schedule": {
+					Type:        schema.TypeString,
+					Required:    true,
+					Description: "The schedule for penetration scan.",
+				},
+				"conformance_scan_schedule": {
+					Type:        schema.TypeString,
+					Required:    true,
+					Description: "The schedule for conformance scan.",
+				},
+			},
+		},
```

Comment:
@nikchern Are all of these expecting a string in front syntax? Because so, we should specify that.